core/include/traccc/utils/propagation.hpp

```diff
@@ -14,6 +14,8 @@
 #include <detray/navigation/direct_navigator.hpp>
 #include <detray/navigation/navigator.hpp>
 #include <detray/propagator/actors.hpp>
+// TODO: Remove when updating to future detray version.
```

Comment:
I think this include can stay that way, since the constrained step size is actually used in multiple other places in the propagation in traccc 
```suggestion
```